packages/replay-internal/package.json

```diff
@@ -73,7 +73,10 @@
     "@sentry-internal/rrweb": "2.15.0",
     "@sentry-internal/rrweb-snapshot": "2.15.0",
     "fflate": "^0.8.1",
-    "jsdom-worker": "^0.2.1"
+    "jest-matcher-utils": "^29.0.0",
```

Comment:
Our custom matchers use a util fn from this lib to pretty print diffs.

Reply:
I guess we can possibly refactor this later to avoid this dependency, but all good for now!